src/test/java/com/ericsson/eiffel/remrem/publish/helper/RMQHelperUnitTest.java

```diff
@@ -92,6 +92,19 @@ private void cleanProperties() {
     	assertTrue(rmqHelper.getHost().equals(host));
     }
 
+    @Test public void testConnection() {
+        try {
+            assertNotNull(rmqHelper.rabbitConnection);
+            rmqHelper.cleanUp();
+            assertNull(rmqHelper.rabbitConnection);
+            rmqHelper.createConnection();
+            assertNotNull(rmqHelper.rabbitConnection);
+        } catch (Exception e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
```

Comment:
If you get an exception this test case will suceed. You should add a fail() here.